EUR-Lex.js

```diff
@@ -103,96 +135,125 @@ function doWeb(doc, url) {
 			for (var i in items) {
 				articles.push(i);
 			}
-			ZU.processDocuments(articles, scrape);
+			return ZU.processDocuments(articles, scrape);
```

Comment:
undo (instead replace the `return true;` from some lines above with just `return;`)